stdlib/multiprocessing/synchronize.pyi

```diff
@@ -28,8 +29,11 @@ class Condition(AbstractContextManager[bool]):
     def wait_for(self, predicate: Callable[[], bool], timeout: float | None = ...) -> bool: ...
     def acquire(self, block: bool = ..., timeout: float | None = ...) -> bool: ...
     def release(self) -> None: ...
+    def __exit__(
+        self, exc_type: type[BaseException] | None, exc_val: BaseException | None, exc_tb: TracebackType | None
+    ) -> bool | None: ...
 
-class Event(AbstractContextManager[bool]):
```

Comment:
This class doesn't seem to be a context manager at runtime, and doesn't seem to have ever been a context manager at runtime.